static/src/javascripts/projects/commercial/modules/messenger/click.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { trackNativeAdLinkClick as _trackNativeAdLinkClick } from '../../../common/modules/analytics/google';
+import { _ } from './click';
+
+const { sendClick } = _;
+
+jest.mock('../../../common/modules/analytics/google', () => ({
+	trackNativeAdLinkClick: jest.fn(),
+}));
+
+const trackNativeAdLinkClick = _trackNativeAdLinkClick as jest.Mock;
```

Comment:
I've seen this pattern quite a lot in the test code of of reassigning `_something` to `something` and I'm 76.89% sure it's not necessary.

We should be able to:

```
import { trackClick } from './blah'

jest.mock('./blah')

(trackClick as jest.Mock).[mockImplementation | mockImplementationOnce | mockReturnValue]()
```

Or if we need to just mock one particular function then:

```
jest.mock('./blah', () => ({
  ...jest.requireActual('./blah'),
  trackClick: jest.fn(),
}));
``` 

I might be wrong :)

Reply:
Good point. I put the renaming line in to avoid having to assert `as jest.Mock` throughout the tests, but on reflection this may actually be the more readable thing to do.